frontend/src/pages/Rider/Schedule.tsx

```diff
@@ -21,13 +22,17 @@ const Schedule = () => {
   const [rides, setRides] = useState<Ride[]>([]);
   const [currRides, setCurrRides] = useState<Ride[]>([]);
   const [pastRides, setPastRides] = useState<Ride[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
   const { id, user } = useContext(AuthContext);
   const { withDefaults } = useReq();
   document.title = 'Schedule - Carriage';
   const refreshRides = useCallback(() => {
     fetch(`/api/rides?rider=${id}`, withDefaults())
       .then((res) => res.json())
-      .then(({ data }) => componentMounted.current && setRides([...data]));
+      .then(({ data }) => {
+        componentMounted.current && setRides([...data]);
```

Comment:
This line seems to be resulting in an "Unhandled Rejection (TypeError): data is not iterable" error when I open the schedule page as a student.